community-expectations.md

```diff
@@ -0,0 +1,85 @@
+## Community Expectations
+
+Kubernetes is a community project and, as such, it is wholly dependent on
+its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration
+technology that radically simplifies the process of creating reliable
+distributed systems, however a second, equally important goal is the creation
+of a community that fosters easy, agile development of such an orcheastration
+systems.
+
+This document describes the community expectations for
```

Comment:
I think it's critically important that the first sentence here needs to be a royal we - i.e. "We the community therefore describe our expectations for our members."   It can't be dispassionate outsider.